OrbitLinuxTracing/Utils.cpp

```diff
@@ -33,11 +33,12 @@ std::string ReadMaps(pid_t pid) {
   }
 }
 
-std::string ExecuteCommand(const std::string& cmd) {
-  std::unique_ptr<FILE, decltype(&pclose)> pipe(popen(cmd.c_str(), "r"),
-                                                pclose);
+std::optional<std::string> ExecuteCommand(const std::string& cmd) {
+  std::unique_ptr<FILE, decltype(&pclose)> pipe{popen(cmd.c_str(), "r"),
+                                                pclose};
   if (!pipe) {
     ERROR("Could not open pipe for \"%s\"", cmd.c_str());
+    return std::optional<std::string>{};
```

Comment:
I thought you discovered this because you added a test for it.. but there seems to be only smoke test, am I missing something?

Reply:
Would be nice to have this test.

Reply:
Sure, but as I said, it might be more complicated than expected.